### DISCUSSION TITLE
Welcome to bob-plugin-openai-translator Discussions!

### DISCUSSION THREAD

**Discussion by yetone** [yetone/bob-plugin-openai-translator#79] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anniemeow1** [comment]
大佬你好，已正常安装使用，请问使用时候，这个openai的API密钥仍然是按照$0.600 / 1M tokens这个计费的吗？还是插件可无视openai的计费呢？期待回复！再次谢谢大佬！
